R/reportEmi.R

```diff
@@ -2536,10 +2535,9 @@ if (!is.null(vm_plasticsCarbon)) {
 
   # 8. Ad-hoc fix for emissions w/o non-energy use and Aggregation to global and regional values  ----
 
-if (is.null(vm_demFENonEnergySector) && (module2realisation["industry", 2] == "fixed_shares")) {
- # (Note: The non-energy use variables are so far only available for REMIND-EU runs and industry fixed_shares)
+
+  # (Note: The non-energy use variables are so far only available for REMIND-EU runs and industry fixed_shares)
   # TODO: add non-energy use variables for all regionmappings and sector realizations
-  #Note (SM): I'm not sure if I got these notes so I created the condition above to try to make sure that this will work anyways
 
   # Note: Non-energy use emissions should not be confused with process emissions. Non-energy use emissions are emissions/carbon flow of FE carriers which are used as feedstocks in industry.
   if ("FE|Non-energy Use|Industry (EJ/yr)" %in% getNames(output) &&
```

Comment:
What would be important is that the version with ``vm_demFENonEnergySector`` does not enter this bracket. We do not need emissions variables corrected for non-energy use in Simon's implementation anymore. However, it looks like it does not enter here because Simon named the variables slightly differently e.g. ``FE|Non-energy Use|+|Industry (EJ/yr)`` instead of ``FE|Non-energy Use|Industry (EJ/yr)``. However, for clarity of the code it would advisable to add a ``is.null(vm_demFENonEnergySector)`` condition.